GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -281,6 +281,7 @@ internal enum Commands
             FocusCommitMessage,
             ResetSelectedFiles,
             StageSelectedFile,
+            StageAll,
```

Comment:
Hotkeys are identified by theirs position in the enumaration. You have to add a new item at the end, otherwise you change the index of the items comming after the inserted item.